hadoop-plugin-test/expectedOutput/negative/missingFields.out

```diff
@@ -12,6 +12,27 @@ RequiredFieldsChecker ERROR: PigJob job9 must set script
 RequiredFieldsChecker ERROR: SparkJob job10 must set uses and executes
 RequiredFieldsChecker ERROR: VoldemortBuildPushJob job11 has the following required fields: storeName, clusterName, buildInputPath, buildOutputPath, storeOwners, storeDesc
 RequiredFieldsChecker ERROR: HadoopShellJob job12 must set command or commands
+RequiredFieldsChecker ERROR: HdfsToTeradataJob job13 has the following required fields: hostName
```

Comment:
This comment no longer applies - you've explained above that GobblinJob won't have additional checks